.github/workflows/pr_and_main.yml

```
@@ -24,7 +24,6 @@ jobs:
         - {os: ubuntu-20.04, python-version: "3.10", build-config: ci-linux}
```

```suggestion
        - {os: ubuntu-22.04, python-version: "3.10", build-config: ci-linux}
```
It's `22.04` everywhere else.